build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Can't check it here in the PR. But please make sure it is executable.
`chmod +x`